docker-compose.yml

```diff
@@ -1,38 +1,67 @@
 
 version: '2'
 services:
+
   postgres:
     container_name: postgres-it
-    image: sdcplatform/ras-rm-docker-postgres
+    image: eu.gcr.io/ons-rasrmbs-management/postgres
+    command: postgres -c 'max_connections=1000'
```

Comment:
Inelegant, but this fixes the `Too many clients already` error. That error appears to be due to failsafe generating multiple connections per test and not purging or reusing the existing connections between tests.